website/source/api/acl-tokens.html.md

```diff
@@ -70,6 +70,10 @@ The table below shows this endpoint's support for
 | ---------------- | ----------------- | ------------ |
 | `YES`            | `all`             | `management` |
 
+### Parameters
+
+- `prefix` `(string: "")` - Specifies a string to filter ACL tokens based on an
```

Comment:
this prefix must have an even number of characters. e.g., "3d" or "3da2" or "3da2ed" or "3da2ed52" (as in the example), or you get the 500 error `acl token lookup failed: index error: Input (without hyphens) must be even length` (because it is decoded to bytes, each byte requiring 2 hex digits)

is this worth noting in the docs?

Reply:
Yeah, it makes sense to have that. I basically copied the text over from the [allocations endpoint](https://github.com/hashicorp/nomad/blob/master/website/source/api/allocations.html.md#parameters) 😬 

I will add this info to the other endpoints as well.

Reply:
Hum...the `prefix` can only have hexadecimal characters as well for the same reason. Is this too cryptic @cgbaker @angrycub?

```
The prefix must have an even number of hexadecimal characters (0-9a-f).
```

Reply:
nope. it's just true. :)

Reply:
What about "Because the value is decoded to bytes, the prefix must have an even number of hexadecimal characters (0-9a-f)."  I do think that the reasoning helps drive the necessity home.

Reply:
Perfect, I will use that. Thanks!

Reply:
I think I got all endpoints that have the `prefix` param. I also clarified which field will be used to match the prefix and fixed some of the examples.